app/models/concerns/outgoings_summarisable.rb

```diff
@@ -1,27 +1,39 @@
 module OutgoingsSummarisable
   def client_mortgage
-    net_housing_costs if client_income_relevant? && property_owned_with_mortgage?
+    return if asylum_support? || !client_income_relevant?
+
+    property_owned_with_mortgage? ? net_housing_costs : 0
   end
 
   def client_rent
-    net_housing_costs if client_income_relevant? && !owns_property?
+    return if asylum_support? || !client_income_relevant?
```

Comment:
This is a bit redundant - `client_income_relevant?` already incorporates a check for `asylum_support`